src/canvas-view/examples/canvas.js

```diff
@@ -0,0 +1,334 @@
+ *         </ul>
+ *     </ul>
+ * </ul>
+ * @param {object} chartViewModel (Optional) The chartViewModel is initialized from the chartDataModel and contains additional helper methods such as <code>chartViewModel.isOnlyOneNodeSelected()</code> and
+ * <code>chartViewModel.getSelectedNodes()</code>.  You only need to specify a chartViewModel object if you plan on using advanced canvas operations.
+ * @param {boolean} readOnly A flag indicating whether the canvas is in 'read-only' mode.  When in 'read-only' mode nodes cannot be moved, selected, or deleted, and the node action toolbar is hidden.
+ * @param {boolean} hideConnectors A flag indicating whether connections should be hidden or shown on the canvas
+ * @example
+ <example module="patternfly.canvas.demo">
+ <file name="index.html">
+   <style>
+     .canvas {
+         background-image: url('/img/canvas-dot-grid.png');
+         background-repeat: repeat;
+     }
+     .canvas-demo-container {
+         height: 600px;
+         margin-left: 15px;
+         overflow: auto;
+         width: 99%;
+     }
+   </style>
+   <div ng-controller="CanvasDemoCtrl" class="example-container">
+     <div class="canvas-demo-container">
+       <pf-canvas chart-data-model="chartDataModel"
+                  chart-view-model="chartViewModel"
+                  read-only="readOnly"
+                  hide-connectors="hideConnectors">
+       </pf-canvas>
+     </div>
+     <hr>
+     <div class="form-group">
+       <label class="checkbox-inline">
+         <input type="checkbox" ng-model="readOnly">Read Only</input>
+       </label>
+
+       <label class="checkbox-inline">
+         <input type="checkbox" ng-model="hideConnectors">Hide Connections</input>
+       </label>
+
+       <button ng-click="addNode()" style="margin-left: 10px;">Add Node</button>
+
+       <button ng-click="selectAll()" style="margin-left: 10px;">Select All (Ctrl+A)</button>
+       <button ng-click="deselectAll()">Deselect All (esc key)</button>
+       <button ng-click="deleteSelected()">Delete Selected (delete key)</button>
+
+       <button ng-click="zoomIn()" style="margin-left: 10px;">Zoom In</button>
+       <button ng-click="zoomOut()">Zoom Out</button>
+     </div>
+     <div style="padding-top: 12px;">
+       <label style="font-weight:normal;vertical-align:middle;">Events: </label>
+     </div>
+     <div>
+       <textarea rows="10" class="col-md-12">{{eventText}}</textarea>
+     </div>
+   </div>
+ </file>
+
+ <file name="modules.js">
+   angular.module('patternfly.canvas.demo', ['patternfly.canvas']);
+ </file>
+
+ <file name="script.js">
+ angular.module( 'patternfly.canvas.demo' ).controller( 'CanvasDemoCtrl', function( $scope ) {
+     $scope.chartDataModel = {
+          "nodes": [
+            {
+              "name": "Nuage",
+              "x": 345,
+              "y": 67,
+              "id": 1,
+              "image": "/img/OpenShift-logo.svg",
+              "width": 150,
```

Comment:
Understood. My suggestion was to use generic items rather than storage specific items ie 'Dolor' rather than 'OpenShift'